celements-spring-mvc/pom.xml

```diff
@@ -24,13 +24,26 @@
   <parent>
     <groupId>com.celements</groupId>
     <artifactId>celements</artifactId>
-    <version>6.2</version>
+    <version>6.3-SNAPSHOT</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <artifactId>celements-spring-mvc</artifactId>
   <version>6.2-SNAPSHOT</version>
   <description>Celements Spring MVC</description>
   <dependencies>
+    <dependency>
+      <groupId>com.celements</groupId>
+      <artifactId>celements-model</artifactId>
+      <version>6.3</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.celements</groupId>
+      <artifactId>celements-core</artifactId>
+      <version>6.12</version>
+      <scope>provided</scope>
+    </dependency>
```

Comment:
afais celements-model/core dependencies are only needed because of the keycloak implementation. i strongly recommend extracting this to its own pom in order to avoid making celements-spring-mvc dependent on celements-model/core